source/_layouts/master.blade.php

```diff
@@ -25,15 +25,19 @@
         <meta name="msapplication-TileImage" content="/mstile-144x144.png">
         <meta name="theme-color" content="#ffffff">
 
-        <link rel="stylesheet" href="{{ $page->asset_prefix }}/css/main.css">
+        <link rel="stylesheet" href="{{ $page->asset_prefix }}/assets/css/main.css">
         <link href='https://fonts.googleapis.com/css?family=Lato:100,300,300italic,400,400italic,500,500italic,700,700italic' rel='stylesheet' type='text/css'>
     </head>
-    <body class="border-t-3 border-primary full-height flex-col">
-        <div class="flex-fit">
-            @yield('body')
-        </div>
-        <footer class="bg-dark text-center p-xs-y-6 p-xs-x-4 text-light-muted text-sm">
-            <p>&copy; {{ date('Y') }} <a href="https://tighten.co" class="link-light-muted">Tighten</a></p>
+    <body class="min-h-screenfont-sans">
```

Comment:
This class doesn't seem right

Reply:
Yeah, it should be `min-h-screen font-sans`, it was fixed in #34 [here](https://github.com/tightenco/jigsaw-site/blob/8004d5e7004e0bb782e388eccf783226ada96d19/source/_layouts/master.blade.php#L31)